platform/lib/sql-proxy.ts

```diff
@@ -0,0 +1,51 @@
+import { ConfigurationError } from "@pipedream/platform";
```

Comment:
Import locally instead of from package

```suggestion
import { ConfigurationError } from "./errors";
```